<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<title>Publications - Huang Lab @ HKU</title>
<meta name="description" content="">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
<link rel="shortcut icon" href="/assets/images/favicon.png">
<!-- CSS -->
<link rel="stylesheet" href="/assets/css/style.css">

<!-- Js -->
<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js" integrity="sha384-SlE991lGASHoBfWbelyBPLsUlwY1GwNDJo3jSJO04KZ33K2bwfV9YBauFfnzvynJ" crossorigin="anonymous"></script>
<script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>


 
  </head>
  
  <body>
    <button onclick="topFunction()" id="scrollbutton" title="Go to top"><span>Top</span></button>
    <header id="top">
    <div class="container">
  
        <div class="text-right">
        <ul class="social-icon">
    
    
    <li class="socials__item">
      <a href="https://github.com/statbiomed" target="_blank" class="socials__item__link" title="Github">
        <i class="fab fa-github" aria-hidden="true"></i>
      </a>
    </li>
    
    
    
    <li class="socials__item">
      <a href="https://twitter.com/YuanhuaHuang" target="_blank" class="socials__item__link" title="Twitter">
        <i class="fab fa-twitter" aria-hidden="true"></i>
      </a>
    </li>
    
    
    
    <li class="socials__item">
      <a href="https://scholar.google.com/citations?user=LUUIw_EAAAAJ&h" target="_blank" class="socials__item__link" title="Google">
        <i class="fab fa-google" aria-hidden="true"></i>
      </a>
    </li>
    
    
  </ul> 
      </div>
    
</div>
</header>
 
<header class="header">
    <nav class="navbar navbar-expand-lg navbar-light">

    
	    <a href="/" class="header-logo">
			<img src="/assets/images/logo.png" class="header__logo__img" width="50px">
		  </a>
       
      <div class="mr-auto">
        <a href="/" class="navbar-title">
          Huang Lab @ HKU
        </a>
      </div>

      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navbarToggler">
        <ul id="nav-link" class="navbar-nav ml-auto">
        
         <li>
         <a href="/" >
        Home
         </a>
        </li>
        
         <li>
         <a href="/team.html" >
        Team
         </a>
        </li>
        
         <li>
         <a href="/research.html" >
        Research
         </a>
        </li>
        
         <li>
         <a href="/publications.html"  style="color:rgb(49, 117, 91)">
        Publications
         </a>
        </li>
        
         <li>
         <a href="/join.html" >
        Join
         </a>
        </li>
        
         <li>
         <a href="/blog.html" >
        Blog
         </a>
        </li>
        
         <li>
         <a href="/contact.html" >
        Contact
         </a>
        </li>
        
       </ul>
      </div>
      
    </nav>

  </header>

        
    <div class="jumbotron global-header">

</div>
  

   
<div class="page-markdown text-left mx-auto">
     
            <p>See the <a href="https://scholar.google.com/citations?user=LUUIw_EAAAAJ&amp;hl=en">Google Scholar page</a> for more collaborative publications.<br /></p>

<h4 id="preprint">Preprint</h4>
<ul>
  <li><a href="https://www.biorxiv.org/content/10.1101/2022.08.19.504616v1">SpatialDM: Rapid identification of spatially co-expressed ligand-receptor reveals cell-cell communication patterns.</a><br />
Li Zhuoxuan, Wang Tianjie, Liu Pengtao# and Huang Yuanhua#<br />
<strong>bioRxiv</strong> 504616 (2022) | <a href="https://github.com/StatBiomed/SpatialDM">Python package</a></li>
  <li><a href="https://www.biorxiv.org/content/10.1101/2022.03.21.485232v1">DCATS: differential composition analysis for complex single-cell experimental designs.</a><br />
<em>Lin, X.</em>, Chau, C., <em>Huang, Y.</em>#, &amp; Ho, J.W.K.#<br />
<strong>bioRxiv</strong> 485232 (2022)</li>
</ul>

<h4 id="2022">2022</h4>
<ul>
  <li><a href="https://www.nature.com/articles/s41467-022-34188-7">UniTVelo: temporally unified RNA velocity reinforces single-cell trajectory inference.</a><br />
Gao Mingze, Qiao Chen and Huang Yuanhua<br />
<strong>Nature Communications</strong> 13, 6586 (2022) | <a href="https://github.com/StatBiomed/UniTVelo">Python package</a> | <a href="https://www.biorxiv.org/content/10.1101/2022.04.27.489808v1">bioRxiv</a></li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btac321">Genomic sequences and RNA binding proteins predict RNA splicing kinetics in various single-cell contexts.</a><br />
Hou Ruiyan and Huang Yuanhua<br /> 
<strong>Bioinformatics</strong>, btac321 (2022) | <a href="https://github.com/StatBiomed/scRNA-efficiency-prediction">Scripts</a> | <a href="https://www.biorxiv.org/content/10.1101/2021.05.02.442314v1">bioRxiv</a></li>
  <li><a href="https://www.nature.com/articles/s41467-022-28845-0">MQuad enables clonal substructure discovery using single cell mitochondrial variants.</a><br />
<em>Kwok A.W.C.</em>, <em>Qiao C</em>, <em>Huang R</em>, Sham M.H., Ho J.W.K.#, <em>Huang Y</em>#.<br />
<strong>Nature Communications</strong> 13, 1205 (2022) | <a href="https://github.com/single-cell-genetics/MQuad">Python package</a> | <a href="https://www.biorxiv.org/content/10.1101/2021.03.27.437331v1">bioRxiv</a></li>
</ul>

<h4 id="2021">2021</h4>
<ul>
  <li><a href="https://www.pnas.org/content/118/49/e2105859118">Representation learning of RNA velocity reveals robust cell transitions.</a><br />
Qiao Chen and Huang Yuanhua<br />
<strong>Proc. Natl. Acad. Sci. U.S.A.</strong> 118(49), e2105859118 (2021) | <a href="https://github.com/qiaochen/VeloAE">Python package</a> | <a href="https://www.biorxiv.org/content/10.1101/2021.03.19.436127v1">bioRxiv</a></li>
  <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02461-5">BRIE2: computational identification of splicing phenotypes from single-cell 
transcriptomic experiments.</a><br />
<em>Huang Y</em>, Sanguinetti G.<br />
<strong>Genome Biology</strong> 22, 251 (2021) | 
<a href="https://github.com/huangyh09/brie">Python package</a> |
<a href="https://brie.readthedocs.io">Manual</a> | 
<a href="https://github.com/huangyh09/brie-tutorials">Tutorials</a></li>
  <li><a href="https://www.sciencedirect.com/science/article/abs/pii/S245231002100069X">Uncertainty vs variability: Bayesian methods for analysis of scRNA-seq data.</a><br />
<em>Huang Y</em>, Sanguinetti G.<br />
<strong>Current Opinion in Systems Biology</strong> 100375 (2021) | review paper</li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btab358">Cellsnp-lite: an efficient tool for genotyping single cells.</a><br />
Huang Xianjie and Huang Yuanhua<br />
<strong>Bioinformatics</strong>, 37(23), 4569-4571 (2021) 
| <a href="https://www.biorxiv.org/content/10.1101/2020.12.31.424913v1">bioRxiv</a> 
| <a href="https://cellsnp-lite.readthedocs.io">software</a></li>
</ul>

<h4 id="2020">2020</h4>
<ul>
  <li><a href="https://doi.org/10.1016/j.stem.2020.07.019">Naive pluripotent stem cells exhibit phenotypic variability that is driven by 
genetic variation.</a><br />
Ortmann D., Brown S., Czechanski A., Aydin S., Muraro D., <em>Huang Y.</em>, 
Tomaz R.A., Osnato A., Canu G., Wesley B.T., Skelly D., Stegle O., …, 
Reinholdt L.G., Vallier L.<br />
<strong>Cell Stem Cell</strong>, 2020, 7(3), pp.470-481.</li>
  <li><a href="https://www.nature.com/articles/s41592-020-0766-3">Cardelino: computational integration of somatic clonal substructure and 
single-cell transcriptomes.</a><br />
McCarthy DJ * , Rostom R * , <em>Huang Y</em> * , Kunz DJ, Danecek P, Bonder MJ, Hagai T, 
  Lyu R., Wang W, Gaffney DJ, Simons BD, Stegle O, Teichmann SA. 
  (*: Joint first author)<br />
<strong>Nature Methods</strong> Volume 17 (2020) p.414–421 | 
<a href="https://github.com/single-cell-genetics/cardelino">R package</a> |
<a href="https://htmlpreview.github.io/?https://github.com/single-cell-genetics/cardelino/blob/master/inst/doc/vignette-cloneid.html">Usage examples</a></li>
</ul>

<h4 id="2019-and-before">2019 and before</h4>
<ul>
  <li><a href="http://dx.doi.org/10.1186/s13059-019-1865-2">Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without 
genotype reference.</a><br />
<em>Huang Y</em>, McCarthy DJ, Stegle O.<br />
<strong>Genome Biology</strong> Volume 20 (2019) p273 |
<a href="https://github.com/single-cell-genetics/vireo">Python package</a> |
<a href="https://vireoSNP.readthedocs.io">Manual</a></li>
  <li><a href="https://link.springer.com/protocol/10.1007/978-1-4939-9057-3_12">Using BRIE to Detect and Analyze Splicing Isoforms in scRNA-Seq Data.
</a><br />
<em>Huang Y</em>, Sanguinetti G.<br />
<strong>Methods Mol Biol</strong> Volume 1935 (2019) p.175-185</li>
  <li><a href="https://doi.org/10.1093/bioinformatics/bty1033">Systematic characterization and prediction of post-translational modification
cross-talk between proteins.</a><br />
Huang R, <em>Huang Y</em>, Guo Y, Ji S, Lu M, Li T.<br />
<strong>Bioinformatics</strong> Volume 35 (2019) p.2626-2633</li>
  <li><a href="http://www.genome.org/cgi/doi/10.1101/gr.225615.117">Transcription rate strongly affects splicing fidelity and cotranscriptionality 
in budding yeast.</a><br />
Aslanzadeh V, <em>Huang Y</em>, Sanguinetti G, Beggs JD.<br />
<strong>Genome Research</strong> Volume 28 (2018) p.203-213</li>
  <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1248-5">BRIE: transcriptome-wide splicing quantification in single cells.
</a><br />
<em>Huang Y</em>, Sanguinetti G.<br />
<strong>Genome Biololgy</strong> Volume 18 (2017) p.123 | 
<a href="https://github.com/huangyh09/brie">Python package</a> |
<a href="https://brie.readthedocs.io">Manual</a></li>
  <li><a href="https://doi.org/10.1093/bioinformatics/btw364">Statistical modeling of isoform splicing dynamics from RNA-seq time series 
data.</a><br />
<em>Huang Y</em>, Sanguinetti G.<br />
<strong>Bioinformatics</strong> Volume 32 (2016) p.2965-2972</li>
  <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0848-1">Transcriptome-wide RNA processing kinetics revealed using extremely short 4tU 
labeling.</a><br />
Barrass JD, Reid JE, <em>Huang Y</em>, Hector RD, Sanguinetti G, Beggs JD, Granneman S. (*: Joint first author)<br />
<strong>Genome Biology</strong> Volume 16 (2015) p.282</li>
  <li><a href="https://doi.org/10.1074/mcp.M114.037994">Systematic characterization and prediction of post-translational modification 
cross-talk.</a><br />
<em>Huang Y</em>, Xu B, Zhou X, Li Y, Lu M, Jiang R, Li T.<br />
<strong>Mol Cell Proteomics</strong> Volume 14 (2015) p.761-770</li>
</ul>


        
</div>

    



      <footer class="footer text-center">
   
        <div class="copyright">
          <span>© 2022 Huang Lab @ HKU</span>
          <span> - proudly designed & developed by </span> WZ Design
          <!-- <a href="https://weizhudesign.github.io/" target="_blank">WZ Design</a> -->
        </div>
        
   
</footer>

  
    <script src="/assets/js/button.js"></script>
  </body> 
  
</html>