<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<title>Research - Huang Lab @ HKU</title>
<meta name="description" content="">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
<link rel="shortcut icon" href="/assets/images/favicon.png">
<!-- CSS -->
<link rel="stylesheet" href="/assets/css/style.css">

<!-- Js -->
<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js" integrity="sha384-SlE991lGASHoBfWbelyBPLsUlwY1GwNDJo3jSJO04KZ33K2bwfV9YBauFfnzvynJ" crossorigin="anonymous"></script>
<script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>


 
  </head>
  
  <body>
    <button onclick="topFunction()" id="scrollbutton" title="Go to top"><span>Top</span></button>
    <header id="top">
    <div class="container">
  
        <div class="text-right">
        <ul class="social-icon">
    
    
    <li class="socials__item">
      <a href="https://github.com/statbiomed" target="_blank" class="socials__item__link" title="Github">
        <i class="fab fa-github" aria-hidden="true"></i>
      </a>
    </li>
    
    
    
    <li class="socials__item">
      <a href="https://twitter.com/YuanhuaHuang" target="_blank" class="socials__item__link" title="Twitter">
        <i class="fab fa-twitter" aria-hidden="true"></i>
      </a>
    </li>
    
    
    
    <li class="socials__item">
      <a href="https://scholar.google.com/citations?user=LUUIw_EAAAAJ&h" target="_blank" class="socials__item__link" title="Google">
        <i class="fab fa-google" aria-hidden="true"></i>
      </a>
    </li>
    
    
  </ul> 
      </div>
    
</div>
</header>
 
<header class="header">
    <nav class="navbar navbar-expand-lg navbar-light">

    
	    <a href="/" class="header-logo">
			<img src="/assets/images/logo.png" class="header__logo__img" width="50px">
		  </a>
       
      <div class="mr-auto">
        <a href="/" class="navbar-title">
          Huang Lab @ HKU
        </a>
      </div>

      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navbarToggler">
        <ul id="nav-link" class="navbar-nav ml-auto">
        
         <li>
         <a href="/" >
        Home
         </a>
        </li>
        
         <li>
         <a href="/team.html" >
        Team
         </a>
        </li>
        
         <li>
         <a href="/research.html"  style="color:rgb(49, 117, 91)">
        Research
         </a>
        </li>
        
         <li>
         <a href="/publications.html" >
        Publications
         </a>
        </li>
        
         <li>
         <a href="/join.html" >
        Join
         </a>
        </li>
        
         <li>
         <a href="/blog.html" >
        Blog
         </a>
        </li>
        
         <li>
         <a href="/contact.html" >
        Contact
         </a>
        </li>
        
       </ul>
      </div>
      
    </nav>

  </header>

        
    <div class="jumbotron global-header">

</div>
  

   
       <div id="research-tile" class="container text-center">
          <div class="row">  
             <div class="col-md-4 col-sm-4 col-xs-12">
                <a href="#cardelino-info">
                <div id="cardelino-tile"><span>Machine Learning</span></div>
                </a>
              </div>

              <div class="col-md-4 col-sm-4 col-xs-12">
                <a href="#brie-info">
                <div id="brie-tile"><span>Single-cell Genomics</span></div>
                </a>
              </div>

            <div class="col-md-4 col-sm-4 col-xs-12">
              <a href="#vireo-info">
              <div id="vireo-tile"><span>Spatial Transcriptomics</span></div>
              </a>
            </div> 

          </div>
         </div>


    <div class="research-intro text-center">
       <div class="container">
          <h2>Research interests</h2>
            <p align="left">
              We are fascinated by how data science accelerates biomedical science, 
              not only as a tool but also as a way of thinking, right in this era 
              with exponential growth of data.
            </p>
            <p align="left">
              Within our lab, we aim to formulise a range of biomedical challenges 
              into data science problems by designing powerful statistical models 
              and developing efficient algorithms, including Markov chain Monte 
              Carlo and variational inference. Our work is mostly problem-driven 
              and methodology focused, and we benefit greatly from close collaborations 
              with experimental labs. Currently, we have a focus on the following themes.
            </p>
            <p align="left">
              You can have a look at our continuous reading in our 
              <a href="https://github.com/StatBiomed/HLab-pubmon" style="color: #5dade2;" target="_blank">paper monitoring</a> 
              archive. 
            </p>
            <p align="left">
              We also activately host  
              <a href="https://statbiomed.github.io/HKUbioinfoNet/" style="color: #5dade2;" target="_blank">HKU cross-lab Bioinformatics Seminar</a> (monthly). 
            </p>
              </div>
            </div>  
          
      
        <div id="cardelino-info">
          <div class="container">
          <div class="row">
            <div class="col-md-12 col-sm-12 col-xs-12">
                <h4>Machine learning methods and their biomedical applications</h4>
                <p>
                  Our recent work covers both supervised learning and unsupervised learning, including 
                  predicting RNA splicing efficiency with genomic sequences and learning effective 
                  embeddings for robust RNA velocity analysis. We are further stretching into deep learning 
                  frameworks, often in conjunction with Bayesian graphical models.
                </p>
                <div class="text-center">
                  <img id="research-img" width="75%" src="/assets/images/research/veloAE_fig.png" class="img-responsive">
                </div>
              </div>
            </div>
          </div>
        </div>
        
        <div id="brie-info">
          <div class="container">
          <div class="row">
            <div class="col-md-12 col-sm-12 col-xs-12">
                <h4>Statistical modelling of single-cell genomics and spatial transcriptomics</h4>
                <p>
                  In the past few years, we have developed a set of statistical methods for analysing 
                  single-cell genomic/transcriptomic data, for example, BRIE (Bayesian Regression for 
                  Isoform Estimate) for RNA splicing quantification and phenotype detection with 
                  efficient variational inference. Currently, we are actively investigating the 
                  dynamics of single-cell transcriptomes and inferring the underlying cell 
                  differentiation, particularly by leveraging RNA velocity, a recent technique based 
                  on intrinsic RNA splicing processes. The recent accumulation of high-quality spatial 
                  transcriptomic data is also promising for dissecting complex systems, and we are 
                  expanding our modelling of cellular transitions through both time and space.
                </p>
                <div class="text-center">
                  <img id="research-img" width="75%" src="/assets/images/research/bayesian_fig.jpg" class="img-responsive">
                </div>
                </div>
            </div>
          </div>
        </div>

        

        <div id="vireo-info">
          <div class="container">
           <div class="row">
            <div class="col-md-12 col-sm-12 col-xs-12">
                <h4>Integrative analysis of cancer mutations and clonal evolution</h4>
                <p>
                  The complexity of cancer tissues is partly caused by the heterogeneity at 
                  multiple molecular layers, e.g., DNA and gene expression, hence integrative 
                  analysis is often crucial for understanding the clonal mutations. We recently 
                  developed a suite of tools for achieving accurate analysis in different steps, 
                  including cellsnp-lite for efficient genotyping of a vast amount of cells, 
                  MQuad for accurately identifying clonal informed variants in the mitochondrial 
                  genome, and Cardelino / Vireo to effective clustering of cells by their clonal 
                  mutations. Along this path, we are further developing methods to integrate 
                  information at multiple levels to more accurately infer clonal structure and 
                  mutation evolution, hence deciphering the impact of somatic mutations on 
                  transcriptome phenotypes.
                </p>
                <ul>
                  <li> multiple assays (e.g., scRNA-seq, single-cell DNA-seq, scATAC-seq, bulk exome-seq)</li>
                  <li> multiple mutation types (e.g., SNV, CNV and mtSNV)</li>
                </ul>
                 
                <div class="text-center">
                  <img id="research-img" width="75%" src="/assets/images/research/cardelino-data.png " class="img-responsive">
                </div>
              </div>
              </div>
            </div>
          </div>

    



      <footer class="footer text-center">
   
        <div class="copyright">
          <span>© 2025 Huang Lab @ HKU</span>
          <span> | Updated on 21 July 2025
          <!-- <span> - proudly designed & developed by </span> WZ Design | June 2023 update -->
          <!-- <a href="https://weizhudesign.github.io/" target="_blank">WZ Design</a> -->
        </div>
        
   
</footer>

  
    <script src="/assets/js/button.js"></script>
  </body> 
  
</html>